osscameroon/camerdevs
# Implement ratings post request

This pull request implement  the `POST` `ratings` endpoint, that creates a new rating entry

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-postrges` to run an initialized database
- run  `make run` to launch the api
- then run the command 
```
curl -s -X POST "http://localhost:7000/ratings" -d'{"company_name":"La Mater", "salary": 150000, "city": "Maroua", "seniority": "Seniority", "rating": 4, "comment": "mon commentaire", "job_title": "my job title"}' -H 'Content-Type: application/json'
```
you should see something like this 👇 

```
{"message":"created"}
```

You can see that last element with `curl -s -X GET "http://localhost:7000/ratings?page=1&limit=200" | jq | tail -n 19`

The output should be close to this 👇 
```
 {
      "salary_id": 105,
      "company_id": 102,
      "company_rating_id": 35,
      "rating": 4,
      "salary": 150000,
      "company_name": "La Mater",
      "seniority": "Seniority",
      "comment": "mon commentaire",
      "job_title": "my job title",
      "country": "Cameroon",
      "city": "Maroua",
      "createdat": "2022-02-13T00:00:00Z"
    }
  ],
  "limit": 200,
  "nbHits": 105,
  "offset": 0
}
```
